config.go

```diff
@@ -216,6 +220,8 @@ type config struct {
 	LtcdMode      *btcdConfig     `group:"ltcd" namespace:"ltcd"`
 	LitecoindMode *bitcoindConfig `group:"litecoind" namespace:"litecoind"`
 
+	Extpreimage *extpreimageConfig `group:"Extpreimage" namespace:"extpreimage"`
```

Comment:
[off-topic] is there a reason we need a pointer here? (for an external library or something)

Reply:
I'm just following the convention established by LND when using a struct for config. There may be a good reason for it, but I don't know enough to comment on it.